build/print.go

```diff
@@ -209,9 +209,12 @@ func compactStmt(s1, s2 Expr, isTopLevel bool) bool {
 	if len(s2.Comment().Before) > 0 {
 		return false
 	}
-	if isCall(s1, "load") && isCall(s2, "load") {
-		// Load statements
+	if isLoad(s1) && isLoad(s2) {
+		// Load statements should be compact
 		return true
+	} else if isLoad(s1) || isLoad(s2) {
+		// Load statements should be separated from anything else
+		return false
```

Comment:
Is it needed?
Top-level statements already have empty line.

Reply:
Only in the build mode, in the bzl mode empty lines are preserved by default.